taxi/views.py

```diff
@@ -1,3 +1,18 @@
+from django.http import HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Manufacturer, Car
+
+
+def index(request):
+    number_of_drivers = Driver.objects.all().count()
```

Comment:
It would be best if all requests are written the same way (Driver.objects.count()).

Reply:
Fixed